### ISSUE TITLE
Price error Input not sorted on sorted BAM file

### ISSUE THREAD

**Issue by pkohvaei** [erhard-lab/gedi#15]
Dear Florian,

Here I include the command and the relevant part of the output:

`gedi -e Price -reads test_2.sortedS.bam -genomic homo_grch38109 -prefix price -progress -plot`

```
Error:
java.lang.RuntimeException: Could not run gedi.riboseq.javapipeline.PriceClusterReads@576239a2
Could not run gedi.riboseq.javapipeline.PriceClusterReads@576239a2
Input not sorted: 1+:629089-629119      [1] x2 0|30|0   3pGACATCCGTGGTAGTTGr > 1+:629088-629119 [3] x2 2|29|0   3pGACATCCGTGGTAGTTGr
```
Using Gedi version 1.0.6a .

Thank you

**Comment by florianerhard** [comment]
The bam file is not correctly sorted. Please use bamlist2cit to convert to cit, this should do the trick!
F

**Comment by pkohvaei** [comment]
The next step we used mablist2cit. Now the tool goes till ORF inference and the throws this error:

```
2024-07-19 10:50:40.794 INFO Starting PriceOrfInference ...
2024-07-19 10:50:41.107 INFO Infer ORFs
PRICE
PRICE is an analysis method for Ribo-seq data.

Error:
java.lang.RuntimeException: Could not run gedi.riboseq.javapipeline.PriceOrfInference@1e440333
Could not run gedi.riboseq.javapipeline.PriceOrfInference@1e440333
Exception in parallel iterator thread!
Index 0 out of bounds for length 0

```

Here I include the files the pipeline has created up to the exit point (hope it helps):

![image](https://github.com/user-attachments/assets/cf3b47c0-3ead-4164-83fc-d4b2a6707019)


Thanks a lot.

**Comment by pkohvaei** [comment]
I go ahead and move the second step into a new issue.
